### ISSUE TITLE
Compile error on generated builder code concerning fields from superclass

### ISSUE THREAD

**Issue by hbrands** [helospark/SparkBuilderGenerator#64]
Activate the following two options:

- add method to create builder based on an instance
- Include fields with setter from superclass

With this input
```
public class TestBean {
    private String invisible;
    private Integer count;
    public Integer getCount() {
        return count;
    }
    public void setCount(Integer count) {
        this.count = count;
    }
}
public class DerivedBean extends TestBean {
    private String name;
}
```
generating the builder for DerivedBean including fields name and count (from superclass) the generated builder constructor with parameter DerivedBean does not compile:

```
        private Builder(DerivedBean derivedBean) {
            this.name = derivedBean.name;
            this.count = derivedBean.count; // <-- error: The field TestBean.count is not visible
        }
```

In my opinion, in this case the generated code should look like:

```
        private Builder(DerivedBean derivedBean) {
            this.name = derivedBean.name;
            this.count = derivedBean.getCount(); // <-- use getter here
        }
```

If there is no visible getter for the field, it should probably be omitted.

**Comment by helospark** [comment]
Hello!
Thanks for reporting this, I'll going to fix it next week.

**Comment by helospark** [comment]
Fix released in 0.0.26, please check.

**Comment by hbrands** [comment]
My test was successful with version 0.0.26. Thanks!
